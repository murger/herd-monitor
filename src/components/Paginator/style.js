import styled from 'styled-components';

export const Notice = styled.div`
  width: 100%;
  padding: 32px;
  box-sizing: border-box;
`;
