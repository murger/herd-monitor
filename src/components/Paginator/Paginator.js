import React from 'react';
import { Notice } from './style';

const Transactions = () => (
	<Notice>Paginator</Notice>
);

export default Transactions;
