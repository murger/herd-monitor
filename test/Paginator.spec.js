import React from 'react';
import renderer from 'react-test-renderer';
import Paginator from '../src/components/Paginator';

it('renders the correctly', () => {
  const tree = renderer.create(<Paginator />).toJSON();
  expect(tree).toMatchSnapshot();
});
