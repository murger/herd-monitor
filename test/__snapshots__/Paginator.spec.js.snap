// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the correctly 1`] = `
<div
  className="sc-bdVaJa eCAsEI"
>
  Paginator
</div>
`;
